app/services/foreman_discovery/fact_to_category_resolver.rb

```diff
@@ -101,6 +101,12 @@ def ipmi
     def settings_discovery_fact_prefix(name)
       Setting["discovery_facts_#{name}".to_sym]
     end
+
+    def interface_ips(iface)
```

Comment:
Please ignore these changes, I'll raise additional PR that will handle the `app/services/foreman_discovery/fact_to_category_resolver.rb`